### ISSUE TITLE
Dropdown not annoucing value of dropdown option on up or down arrow key press when dropdown is in collapsed state

### ISSUE THREAD

**Issue by mdmdakbar** [OfficeDev/office-ui-fabric-react#4432]
Repro step:
 Open https://developer.microsoft.com/en-us/fabric#/components/dropdown in Edge browser
Select Basic uncontrolled example
Let the dropdown be in collapsed mode
Press up or down arrow key.
Narrator announces up arrow key or down arrow key only and not the options which are also changed with these key press.

Dropdown in expanded mode works fine

Product affected: VSTS

**Comment by ecraig12345** [comment]
I also noticed this recently when testing a control in SharePoint which uses a dropdown. Definitely needs to be fixed.

**Comment by KevinTCoughlin** [comment]
I wonder if this happens because only the content of the node which renders the option text is changing with UP/DOWN vs. when selected via mouse the outer component re-renders as well as that node's content changing so Narrator picks up on it.

**Comment by ecraig12345** [comment]
Forgot to mention before, I suspect this may be a regression--we've used Fabric dropdowns in various features, and if this issue had always existed, I would have expected it to get caught while we were testing accessibility for some previous feature. So it may be helpful to look through the version history.

**Comment by cliffkoh** [comment]
For some reason, I was unable to repro the fix but #5385 *should* fix it...

**Comment by cliffkoh** [comment]
Fixed in #5385
